lib/runtime/storage/storagediff.go

```diff
@@ -290,3 +296,27 @@ func (cs *storageDiff) applyToTrie(t trie.Trie) {
 
 	}
 }
+
+func (cs *storageDiff) insertSortedKey(key string) {
+	pos := sort.Search(len(cs.sortedKeys), func(i int) bool {
+		return cs.sortedKeys[i] >= key
+	})
```

Comment:
Given that the list is sorted can we use `slice.BinarySearch` https://pkg.go.dev/slices#BinarySearch

Reply:
Yeah I thought that `sort.Search` and `slice.BinarySearch` were equivalent. But I'll change it! thks